src/services/FeedChecker.ts

```diff
@@ -0,0 +1,20 @@
+import {injectable, inject} from 'inversify';
+import {Logger} from 'winston';
+
+@injectable()
+class FeedChecker {
+  @inject('Logger') private logger!: Logger;
+
+  getBaseAndQuote(feedSymbol: string | undefined): string[] {
```

Comment:
```suggestion
  apply(feedSymbol: string | undefined): string[] {
```

Reply:
done

Reply:
I see no change

Reply:
Sorry this file shouldn't be included. I renamed it and accidentally added the new one without deleting this one. Thanks for this.